### PR TITLE
AFHS-571 Use sanitized postcode for validation and request body.

### PR DESCRIPTION
**Pull request by dwybourn**
Updated the manual address validation to validate the sanitised postcode (upper cased and multiple whitespace replaced) instead of the postcode entered by in by the user.

The request body has also been updated to send the sanitised postcode to the claim service.

### CODE REVIEW

File: src/web/routes/application/steps/address/manual-address/test/validation-composition.test.js [hunk 1]
```diff
@@ -72,6 +73,17 @@ test('validation middleware errors for invalid postcode field', async (t) => {
   t.end()
 })
 
+test('validation middleware errors for non UK postcode field', async (t) => {
```
Comment (dwybourn) [inline src/web/routes/application/steps/address/manual-address/test/validation-composition.test.js]: This unit test was missing from when the channel islands regex check was added.

File: src/web/routes/application/steps/address/request-body.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
-const { ADDRESS_KEYS } = require('./constants')
-
-const getAddressFieldFromClaim = (claim) => (address, key) => ({ ...address, [key]: claim[key] })
-
 const requestBody = (session) => ({
-  address: ADDRESS_KEYS.reduce(getAddressFieldFromClaim(session.claim), {})
+  address: {
```
Comment (dwybourn) [inline src/web/routes/application/steps/address/request-body.js]: Previous code was getting all of the address fields as entered, which meant the non sanitised postcode was being added to the request body.

Comment (YetAnotherMatt) [reply src/web/routes/application/steps/address/request-body.js]: Not sure what the answer should be, but should we be using the ADDRESS_KEYS from constants here?

Comment (dwybourn) [reply src/web/routes/application/steps/address/request-body.js]: Updated to use the address constants.

**Pull request merged**